### PR TITLE
feat: thread updates

### PR DESCRIPTION
**Pull request by advaith1**
* https://github.com/discord/discord-api-docs/pull/3502

### CODE REVIEW

File: rest/v9/guild.ts [hunk 1]
```diff
@@ -303,6 +304,11 @@ export type RESTPatchAPIGuildChannelPositionsJSONBody = Array<{
  */
 export type RESTPatchAPIGuildChannelPositionsResult = never;
 
+/**
+ * https://discord.com/developers/docs/resources/guild#list-active-threads
+ */
+export type RESTGetAPIGuildThreadsResult = Omit<APIThreadList, 'has_more'>;
```
Comment (vladfrangu) [inline rest/v9/guild.ts:310]: Can't you just remove the has_more since it's getting yeeted anyways?

Comment (vladfrangu) [inline rest/v9/guild.ts:310]: Just a question since phone review, is this route in the routes object?

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: rest/v9/guild.ts (hunk 1 above)
Comment (advaith1) [reply rest/v9/guild.ts:310]: the get channel (public/private/joined private) archived threads routes are keeping it, it's only removed from get guild active threads

Comment (advaith1) [reply rest/v9/guild.ts:310]: oops, was gonna do it but forgot, will add

File: rest/v9/index.ts [hunk 2]
```diff
@@ -435,9 +435,19 @@ export const Routes = {
...
+	 * - GET `/guilds/{guild.id}/threads/active`
+	 */
+	guildActiveThreads(guildId: Snowflake) {
+		return `/guilds/${guildId}/threads/active` as const;
+	},
+
 	/**
 	 * Route for:
 	 * - GET `/channels/{channel.id}/threads/active`
+	 * 	 (deprecated, use [List Active Guild Threads](https://discord.com/developers/docs/resources/guild#list-active-threads) instead.
```
Comment (vladfrangu) [inline rest/v9/index.ts:449]: Use `@deprecated` instead

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: rest/v9/index.ts (hunk 2 above)
Comment (advaith1) [reply rest/v9/index.ts:449]: didnt seem to render inline, and that might mark the whole function as deprecated while its only deprecated for that particular route

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**